### DISCUSSION TITLE
Docker image out of date

### DISCUSSION THREAD

**Discussion by redimongo** [btcpayserver/btcpayserver#6404] [Technical Support]
So the docker image that is recommended is out of date, its stuck on version _BTCPay Server v1.13.7+83c948d12 (Altcoins)_

Wondering how we update this? We are using this.
<img width="1042" alt="Screenshot 2024-11-16 at 4 40 51 PM" src="https://github.com/user-attachments/assets/0ef5127a-afe5-45c3-ab91-187cdcf39952">

**Comment by dennisreimann** [comment]
See the [v2 blogpost](https://blog.btcpayserver.org/btcpay-server-2-0/#-updating-to-20):

```bash
cd $BTCPAY_BASE_DIRECTORY/btcpayserver-docker
git fetch -a
git checkout 2.0
./btcpay-update.sh
```
